src/main/resources/db/migration/V1_0__Create_User.sql

```diff
@@ -0,0 +1,15 @@
+--------------
+---- User ----
+--------------
+create type "role" as enum ('STUDENT','TEACHER','STAFF');
```

Comment:
STAFF --> MANAGER

Also, check whether the type exists before its creation. either `create type if  not exists...` (or try `drop type if exists...` before creation). The goal is to be able to run the script multiple times but with the same ultimate effect. This is called idempotency

Reply:
But for what purpose would we run the script multiple times if we want to have the same effect ?

Reply:
https://www.red-gate.com/hub/product-learning/flyway/creating-idempotent-ddl-scripts-for-database-migrations#:~:text=The%20solution%3A%20idempotent%20SQL%20migration%20scripts&text=It%20is%20really%20a%20fancy,safely%20into%20a%20Flyway%20migration.